master/buildbot/test/integration/test_integration_secrets_with_vault.py

```diff
@@ -66,9 +69,12 @@ def masterConfig():
             name="force",
             builderNames=["testy"])]
 
+    # note that as of December 2018, the vault docker image default to kv
```

Comment:
should we fix the version of vault in the tests?

Reply:
Looking at https://hub.docker.com/_/vault/ it does not seem that there's some tag with v1 kv secret engine as default.